claat/parser/md/parse.go

```diff
@@ -590,6 +590,16 @@ func code(ds *docState, term bool) types.Node {
 		v = "\n" + v
 	}
 	n := types.NewCodeNode(v, term)
+	for _, attr := range ds.cur.Attr {
+		if attr.Key == "class" {
+			for _, class := range strings.Split(attr.Val, " ") {
+				if strings.HasPrefix(class, "language-") {
+					n.Lang = class
```

Comment:
Wouldn't you want it without the "language-" prefix?

```suggestion
					n.Lang = class[len("language-"):]
```

Reply:
Hi @narphorium thanks for the review and sorry for the very late response. I didn't strip out the prefix here since I noticed here

https://github.com/googlecodelabs/tools/blob/46f2ab2d8b563f6f3134d9dc0ba6245a8c68dfec/claat/render/html.go#L221

`Lang` is added as a class without adding a prefix. The `google-prettify` library expects e.g., `language-java` as the class

https://github.com/google/code-prettify#how-do-i-specify-the-language-of-my-code

I wasn't sure it was appropriate to change the `render` method, mainly because I don't know what the semantics of `language=` should be I'm not aware of any language that respects that attribute.

Let me know if it's better to strip the prefix here, and update render to add it back for the class.

Reply:
/cc @DanSanche